app/src/main/res/navigation/navigation.xml

```diff
@@ -69,7 +69,21 @@
     <action
         android:id="@+id/board"
         app:destination="@id/navbar_board" />
+    <action
+        android:id="@+id/feed"
+        app:destination="@id/navbar_feed" />
```

Comment:
Do we use this action anywhere? If we don't redirect to the feed page from anywhere we don't need this global action.